README.md

```diff
@@ -1,42 +1,177 @@
-# CytoPipe
+# 🐍CytoSnake
 
 "Reproducible pipelines for processing high-dimensional systems morphology data with snakemake 🐍 "
 
-# Table of contents
+## Table of contents
 
-- [CytoPipe](#cytopipe)
-- [Table of contents](#table-of-contents)
-  - [Formatting](#formatting)
-    - [Black](#black)
-    - [Snakefmt](#snakefmt)
-- [Testing](#testing)
-  - [Dry Runs](#dry-runs)
+- [🐍CytoSnake](#cytosnake)
+  - [Table of contents](#table-of-contents)
+  - [About](#about)
+  - [Installation](#installation)
+  - [Workflows](#workflows)
+  - [💻 Development](#-development)
+    - [Pre-commits](#pre-commits)
+      - [how to use it](#how-to-use-it)
+    - [Testing](#testing)
+    - [Dry Runs](#dry-runs)
 
-## Formatting
+## About
 
-### Black
-`Black` is a python code formatter that follows `PEP8` rules in order to maintain source code in a readable fromat.
+CytoSnake is a command line interface (CLI) tool that contains reproducible workflows that analysis high dimensional single-cell morphology datasets.
+CytoSnake's workflows are written in [`Snakemake`](https://github.com/snakemake/snakemake), which is a well established workflow manager that contains important features like data reproducibility, scalability and modularity.
 
-To install `Black`:
+`CytoSnake` make it easy for user to interact as it requires little inputs and parameters. below is an example on how to execute `CytoSnake` once installed
+
+```text
+
+```
+
+## Installation
+
+First, install `CytoSnake` into your local machine:
+
+```text
+git clone https://github.com/WayScience/CytoSnake.git
+
+```
+
+After cloning the repository, go into the `CytoSnake/` directory and create the `CytoSnake` environment
```

Comment:
This will be solved once `CytoSnake` becomes pip installable. Then you can just add `cytosnake` as a dependency in your environment. 

The only downside is that `cytosnake` currently works with python3.10. I do not know if most of your environments run on python3.10

Reply:
Yeah unfortunately when I try to make a "main" environment with CellProfiler I can not use any version above Python 3.8. It will be fine to have two different environments. Thank you for the clarification!